ci-operator/jobs/openshift/origin/openshift-origin-main-presubmits.yaml

```diff
@@ -4453,25 +4457,24 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )e2e-metal-ipi-ovn-ipv6,?($|\s.*)
+    trigger: (?m)^/test( | .* )e2e-metal-ipi-ovn-dualstack-bgp-techpreview,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Why did some of those changed to `false` and some to `true`?

Reply:
No clue I have just call `make jobs`